src/components/hooks/dates.ts

```diff
@@ -41,6 +42,7 @@ import {useLanguagePrefs} from '#/state/preferences'
  */
 const locales: Record<AppLanguage, Locale | undefined> = {
   en: undefined,
+  an,
```

Comment:
```suggestion
  an: undefined,
```
As above, it looks like the `date-fns` package doesn't yet support the `an` locale, so I think this change is needed.